public/app/core/components/AppChrome/ReturnToPrevious/DismissableButton.tsx

```diff
@@ -0,0 +1,58 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { Button, ButtonGroup, useStyles2 } from '@grafana/ui';
+import { t } from 'app/core/internationalization';
+
+export interface DismissableButtonProps {
+  label: string;
+  onClick: () => void;
+  onDismiss: () => void;
+}
+
+export const DismissableButton = ({ label, onClick, onDismiss }: DismissableButtonProps) => {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <ButtonGroup className={styles.buttonGroup}>
+      <Button
+        icon="angle-left"
+        size="sm"
+        variant="primary"
+        fill="outline"
+        onClick={onClick}
+        title={label}
+        className={styles.mainDismissableButton}
+      >
+        {label}
+      </Button>
+      <Button
+        icon="times"
+        aria-label={t('dismissable-button.close', 'Close')}
```

Comment:
Not sure about this name 🤔 but not sure either about using `return-to-previous.dismissable-button.close` as this is a generic component 🤷

Reply:
You may have already considered it but what about `return-to-previous.dismissable-button`? 
Especially the first part `return-to-previous` should be there so both translations (from the button and the label) are grouped together in the translations.

Reply:
Yes, but that is my doubt. As I said, the `DismissableButton` can be used in other areas, so should I add a specific area when it is not actually attached to one?

Reply:
Ah, now I get it. 
The button is built for this feature and it's only in use there. Additionally it's not a component that is officially provided to other features (e.g. not in grafana-ui, not mentioned in any design documentation like Saga). Since I'm not a fan of providing perfect solutions for potential use cases in the future linking it to the feature feels ok to me.